cmake/CMakeLists.txt

```diff
@@ -87,7 +87,7 @@ option(onnxruntime_USE_QNN "Build with QNN support" OFF)
 option(onnxruntime_USE_SNPE "Build with SNPE support" OFF)
 option(onnxruntime_USE_RKNPU "Build with RKNPU support" OFF)
 option(onnxruntime_USE_DNNL "Build with DNNL support" OFF)
-option(onnxruntime_USE_JBLAS "Build MLAS with JBLAS support" ON)
+option(onnxruntime_USE_NEURAL_SPEED "Build with Neural Speed support" ON)
```

Comment:
Just a reminder that there are some known issues from the new dependency: neural-speed. So it's better to turn this option OFF when you build the code.

Reply:
Should I resolve this with a separate PR after this merge goes in?

Reply:
Sure, whatever works for you.